circle.yml

```diff
@@ -1,12 +1,12 @@
 machine:
-  services:
-    - docker
+  python:
+    version: 3.5.2
 
 dependencies:
-  override:
-    - docker info
-    - docker build -t civisanalytics/muffnn .
+  post:
+    - pip install flake8 pytest pytest-faulthandler
```

Comment:
This installs packages needed for dev. See here https://circleci.com/docs/1.0/configuration/